src/components/MultiSelect/MultiSelect.js

```diff
@@ -103,6 +108,7 @@ export default class MultiSelect extends React.Component {
     type: 'default',
     light: false,
     title: false,
+    translateWithId: () => {},
```

Comment:
AFAIK default `translateWithId` in this codebase returns pseudo translation. For `<MultiSelect>`, letting `<ListBox.MenuIcon>` do that seems enough.

Reply:
don't we need to expose this on `<MultiSelect>` to pass through to `<ListBox.MenuIcon>`?

Reply:
If I understand your question correctly - What I mean here is we don't need `defaultProps` here - Passing along `undefined would be fine.

Reply:
got it @asudoh, removed the default function value